### PR TITLE
review of rest api docs

### PR DESCRIPTION
**Pull request by jodygarnett**
Mostly looking for missing query parameters and examples, we will get a short list of missing documentation as well.

### CODE REVIEW

**Comment by jodygarnett** [comment]
@ianturton you did the initial workspace swagger docs can you review my small change here?

File: src/restconfig/src/main/java/org/geoserver/rest/catalog/WorkspaceController.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public RestWrapper workspacesGet() {
     }
 
     @GetMapping(value = "/{workspaceName}")
-    public RestWrapper<WorkspaceInfo> getWorkspace(@PathVariable String workspaceName) {
+    public RestWrapper<WorkspaceInfo> workspaceG(@PathVariable String workspaceName) {
```
Comment (tbarsballe) [inline src/restconfig/src/main/java/org/geoserver/rest/catalog/WorkspaceController.java]: Looks like there is a mistake here?

Comment (jodygarnett) [reply src/restconfig/src/main/java/org/geoserver/rest/catalog/WorkspaceController.java]: Yeah should be workspaceGet to agree with the naming convention; fixing now

**Comment by jodygarnett** [comment]
corrected, will wait on travis build

**Pull request merged**